tests/DataCollector/GuzzleCollectorTest.php

```diff
@@ -74,5 +74,10 @@ public function testCollectCurlData()
             escapeshellarg('http://foo.bar')
         ), $calls[0]['curl']
         );
+
+        $client->post('http://foo.bar', ['body' => str_pad('', GuzzleCollector::MAX_BODY_SIZE + 1)]);
```

Comment:
You could simply initialise the collector with a small body size, but that's fine by me! :)